### DISCUSSION TITLE
Unsupported network!

### DISCUSSION THREAD

**Discussion by developerat79** [hashlips-lab/nft-erc721-collection#359] [Q&A]
When i run the DAPP it says Unsupported network! The contract is deployed on Rinkeby Network

**Comment by ricosannn** [comment]
Same problem here..
You fixed it?

**Comment by biasaurs** [comment]
please someone answer thissss @liarco

**Reply by johncliffalvarez** [reply]
> please someone answer thissss @liarco

I'm having the same issue - Strange thing is that not only does the minting page gives me an the "Error unsupported network" message, but I am still able to connect my wallet... Yet the error message stays up and I don't see neither my wallet address being displayed, nor the minting interface appear to start minting NFTs. Is there something I'm missing? Would greatly appreciate the help!

**Comment by higgsbelly** [comment]
Go the collectionconfig and change the following from hardhatlocal

```const CollectionConfig: CollectionConfigInterface = {
  testnet: Networks.ethereumTestnet,
  mainnet: Networks.ethereumMainnet,```

**Comment by liarco** [comment]
Hi everyone,
I just released a patch version ([2.4.3](https://github.com/hashlips-lab/nft-erc721-collection/releases/tag/2.4.3)) which moves to Goerli as the official testnet for Ethereum and uses the Testnet by default instead of the local Hardhat one.

There is still the option to use Rinkeby as explained in #403 and the changes can be easily applied manually by checking out the changelog from the latest release: https://github.com/hashlips-lab/nft-erc721-collection/compare/2.4.2...2.4.3

Thanks everyone for your support.
